models/nequip/test_nequip_kappa.py

```
@@ -0,0 +1,348 @@
+        freqs_dict = {Key.has_imag_ph_modes: has_imaginary_freqs, Key.ph_freqs: freqs}
+
+        # if conductivity condition is met, calculate fc3
+        ltc_condition = ignore_imaginary_freqs or not has_imaginary_freqs
+
+        if ltc_condition:
+            fc3_set = ltc.calculate_fc3_set(
+                ph3, calculator=calc, pbar_kwargs={"position": task_id}
+            )
+            ph3.produce_fc3(symmetrize_fc3r=True)
+        else:
+            warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)